config-solaris.md

```diff
@@ -0,0 +1,88 @@
+# Solaris Application Container Configuration
+
+Solaris application containers can be configured using the following properties, all of the below properties are the same as given in zonecfg(1M) man page, except milestone. The Solaris specification is entirely optional.
```

Comment:
On Fri, Apr 29, 2016 at 11:06:00AM -0700, Abhijeeth Nuthan wrote:
> Having said that, would it be more appropriate to the the above text
> in config.md as opposed to config-solaris.md and have a entry for
> each platform to say if the platform-specific section is required by
> the said platform.

Yeah, putting something more structured in [1] would be good.  It
looks like you're softening the existing SHOULD to a MAY.  I think
that's also the case for Linux, but I'm not sure, and having each
field listed with our usual:

**`{key}`** *({type}, {when-needed})* {description}

would make it more obvious.

[1]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/config.md#platform-specific-configuration